DOCS/man/mpv.rst

```diff
@@ -203,6 +206,12 @@ Ctrl+s
     Take a screenshot, as the window shows it (with subtitles, OSD, and scaled
     video).
 
+HOME
+    Seek to the beginning of the file.
+
+END
+    Seek to 95% of the file.
```

Comment:
I don't like magic 95% seek. Either make it seek to the real end or not at all.

Reply:
How is seeking to 95% useful? The same case but got to real and and stay there (depending on other mpv options).

Reply:
I just gave an example of how it's useful?? Seek to end would just be quit or playlist-next with the default options.

Reply:
95% in 10 minute clip is quite different than in 3 h movie. For me END means end, not some 5%, but anyway. Not gonna argue that.

Reply:
I think the first thing we need is that END is meaningful, and we can probably agree that it's not meaningful with the default mpv options.

So the question is which value should be used instead.

Few options came up, among them 95%, 99%, someplace in between, or a fixed delta from the end, for instance 1 minute.

I don't think there's a perfect solution here, and between the imperfect options we can choose from, I think we should just pick one. I don't have a personal preference between them, but if someone does, so let's go with that.

Reply:
If we're going to have this, I think some fixed delta from the end is the way to go. Like Kasper said, using a percent is going to be wildly different depending on the length of the clip.

Reply:
Changed to `END seek -60 absolute`.

Reply:
Honestly I would bind only HOME. If anything END could jump to last chapter or do nothing if there are no chapters. Any time related seek for END whether relative or absolute seems wrong.

Reply:
Removed `END`.